pkgs/applications/science/math/R/default.nix

```diff
@@ -49,10 +52,22 @@ stdenv.mkDerivation rec {
       LDFLAGS="-L${gfortran.cc}/lib"
       RANLIB=$(type -p ranlib)
       R_SHELL="${stdenv.shell}"
+  '' + stdenv.lib.optionalString stdenv.isDarwin ''
+      --without-tcltk
```

Comment:
If Tcl/Tk support is disabled, then it shouldn't be passed as a build input either or people will end up compiling that package needlessly.